std/internal/entropy/entropy.d

```diff
@@ -0,0 +1,229 @@
+// Written in the D programming language.
+
+/+
+    Entropy library prototype.
+
+    This code has not been audited.
+    Do not use for cryptographic purposes.
+
+    License:   $(HTTP www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+    Authors:   Elias Batek
+    Source:    $(PHOBOSSRC std/internal/entropy/entropy.d)
+ +/
+module std.internal.entropy.entropy;
+
+public import std.internal.entropy.common;
+
+import std.internal.entropy.bsd;
+import std.internal.entropy.linux;
+import std.internal.entropy.posix;
+import std.internal.entropy.windows;
+import std.meta;
+
+@safe unittest
+{
+    auto buffer = new ubyte[](32);
+    forceEntropySource(defaultEntropySource);
+    const result = getEntropy(buffer);
+
+    assert(
+        !result.isUnavailable,
+        "The default entropy source for the target platform"
+        ~ " is unavailable on this machine. Please consider"
+        ~ " patching it to accommodate to your environment."
+    );
+    assert(result.isOK);
+}
+
+@nogc nothrow:
+
+// Flagship function
+EntropyResult getEntropy(scope void[] buffer) @safe
+{
+    return getEntropyImpl(buffer);
+}
+
+// Convenience overload
+EntropyResult getEntropy(scope ubyte[] buffer) @safe
+{
+    return getEntropy(cast(void[]) buffer);
+}
+
+// Convenience wrapper
+EntropyResult getEntropy(scope void* buffer, size_t length) @system
+{
+    return getEntropy(buffer[0 .. length]);
+}
+
+// Used to manually set the entropy source to use
```

Comment:
````
Public declaration 'forceEntropySource' is undocumented.
```
Use `///`

Reply:
IMHO it doesn’t make much sense to document functionality in `std.internal` anyway.
I’m a bit surprised that I don’t get away with it unlike other modules in said package.
To be honest, I’m tempted to just add exclusions for the files in `dscanner.ini`.

Reply:
At this point, I think I have documented things to an extent that goes far beyond what could be reasonably required for an internal module. I’ll look into D-Scanner warnings that are still left, but I get more and more tempted to add exclusions. (I’ve already added one for the has-public-examples check.)